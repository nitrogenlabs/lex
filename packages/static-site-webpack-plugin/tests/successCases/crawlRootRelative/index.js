module.exports = (locals) => new Promise((resolve) => {
  switch(locals.path) {
    case '/': {
      return resolve(locals.template({
        link: '/foo',
        path: locals.path
      }));
    }
    case '/foo': {
      return resolve(locals.template({
        link: '/foo/bar',
        path: locals.path
      }));
    }
    case '/foo/bar': {
      return resolve(locals.template({
        link: '/',
        path: locals.path
      }));
    }
    default: {
      return resolve('404');
    }
  }
});
