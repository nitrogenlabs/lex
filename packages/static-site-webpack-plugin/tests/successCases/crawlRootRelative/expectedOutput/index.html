<h1>/</h1>

<a href="/foo">Crawl this</a>
